packages/cli/e2e/__tests__/deploy.spec.ts

```diff
@@ -119,7 +119,6 @@ describe('deploy', () => {
       cliVersion: '4.0.9',
     })
     expect(status).toBe(0)
-    expect(stderr).toBe('')
```

Comment:
Please put this back and change the cli version as we discussed to something like `0.0.1-dev` to disable the warning. We will otherwise get this update warning each time we run the cli locally which we don't want